.erb/scripts/notarize.js

```diff
@@ -22,9 +22,11 @@ exports.default = async function notarizeMacos(context) {
   const appName = context.packager.appInfo.productFilename;
 
   await notarize({
+    tool: "notarytool",
     appBundleId: build.appId,
     appPath: `${appOutDir}/${appName}.app`,
     appleId: process.env.APPLE_ID,
     appleIdPassword: process.env.APPLE_ID_PASS,
+    teamId: process.env.APPLE_TEAM_ID
```

Comment:
I would also check for `APPLE_TEAM_ID` and log and error in the if-statement above.